x-pack/plugin/build.gradle

```diff
@@ -13,6 +13,7 @@ archivesBaseName = 'x-pack'
 dependencies {
   testImplementation project(xpackModule('core'))
   testImplementation(testArtifact(project(xpackModule('core'))))
+  testImplementation project(':test:framework')
```

Comment:
I'm wondering if instances like this should really be `testApi` since the intent is that folks that consume this test artifact will also obviously need the test framework to compile against this.

Reply:
yeah this makes kind of sense. I'd like to revisit the api of our testFixtures in a separate commit.

Reply:
FWIW, @dakrone already hit this issue, where he pulled in this change and then had too add a bunch of transitive dependencies to get things working again.